.signadot/testing/e2e-playwright-job.yaml

```diff
@@ -0,0 +1,38 @@
+spec:
+  namePrefix: hotrod-playwright-e2e
+  runnerGroup: playwright
+  script: |
+    #!/bin/bash
+    set -e
+    
+    # Clone the git repo
+    echo "Cloning signadot repo"
+    git clone --single-branch -b "@{branch}" \
+      https://github.com/signadot/hotrod.git
+
+    # Run all playwright tests
+    cd hotrod
+    
+    export HOTROD_NAMESPACE="@{namespace}"
+    export CI=true
+    
+    export SANDBOXED_FRONTEND="@{sandboxed_frontend}"
```

Comment:
Can we remove these vars similar to https://github.com/signadot/hotrod/blob/main/.signadot/testing/demo-tests-job.yaml? Ideally, I'd like to make the test agnostic of sandbox names so that it makes for an easy to understand demo.

Reply:
I created the demo so it's easier to run